cmd/symbols/build.sh

```diff
@@ -1,199 +1,35 @@
-
-    echo "Building the libsqlite3-pcre Docker image..."
-    docker build --progress=plain --quiet -f cmd/symbols/libsqlite3-pcre/Dockerfile -t "libsqlite3-pcre" "$EMPTY_DIRECTORY"
-    echo "Building the libsqlite3-pcre Docker image... done"
-}
-
-# Builds the Docker images that the symbols Docker image depends on. The caller
-# must set:
-#
-# - SYMBOLS_EXECUTABLE_OUTPUT_PATH
-# - CTAGS_D_OUTPUT_PATH
-function buildSymbolsDockerImageDependencies() {
-    if [ -z "$SYMBOLS_EXECUTABLE_OUTPUT_PATH" ]; then
-        echo "buildSymbolsDockerImageDependencies expects SYMBOLS_EXECUTABLE_OUTPUT_PATH to be set."
-        exit 1
-    fi
-    if [ -z "$CTAGS_D_OUTPUT_PATH" ]; then
-        echo "buildSymbolsDockerImageDependencies expects CTAGS_D_OUTPUT_PATH to be set."
-        exit 1
-    fi
-
-    export GO111MODULE=on
-    export GOARCH=amd64
-    export GOOS=linux
-    buildExecutable
-
-    buildCtagsDockerImage
-
-    buildLibsqlite3PcreDockerImage
-
-    cp -R cmd/symbols/.ctags.d "$CTAGS_D_OUTPUT_PATH"
-}
-
-# Builds the symbols Docker image.
-function buildSymbolsDockerImage() {
-    symbolsDockerBuildContext="$(mktemp -d)"
-    trap "rm -rf $symbolsDockerBuildContext" EXIT
-
-    SYMBOLS_EXECUTABLE_OUTPUT_PATH="$symbolsDockerBuildContext/symbols"
-    CTAGS_D_OUTPUT_PATH="$symbolsDockerBuildContext/.ctags.d"
-    buildSymbolsDockerImageDependencies
-
-    echo "Building the $IMAGE Docker image..."
-    docker build --progress=plain --quiet -f cmd/symbols/Dockerfile -t "$IMAGE" "$symbolsDockerBuildContext" \
-        --build-arg COMMIT_SHA \
-        --build-arg DATE \
-        --build-arg VERSION
-    echo "Building the $IMAGE Docker image... done"
-}
-
-# Builds the ctags docker image, used by universal-ctags-dev and the symbols Docker image.
-function buildCtagsDockerImage() {
-    ctagsDockerBuildContext="$(mktemp -d)"
-    trap "rm -rf $ctagsDockerBuildContext" EXIT
-
-    cp -R cmd/symbols/.ctags.d "$ctagsDockerBuildContext"
-
-    echo "Building the $CTAGS_IMAGE Docker image..."
-    docker build --progress=plain --quiet -f cmd/symbols/internal/pkg/ctags/Dockerfile -t "$CTAGS_IMAGE" "$ctagsDockerBuildContext"
-    echo "Building the $CTAGS_IMAGE Docker image... done"
+OUTPUT=`mktemp -d -t sgdockerbuild_XXXXXXX`
+cleanup() {
+    rm -rf "$OUTPUT"
 }
-
-command="$1"
-if type -t "$command" >/dev/null 2>&1; then
-    "$command"
-else
-    echo "Unknown command: $command (must be: execute, buildLibsqlite3Pcre, or buildSymbolsDockerImage)"
-fi
+trap cleanup EXIT
+
+# Environment for building linux binaries
+export GO111MODULE=on
+export GOARCH=amd64
+export GOOS=linux
+export CGO_ENABLED=0
+
+cp -a ./cmd/symbols/.ctags.d "$OUTPUT"
+cp -a ./cmd/symbols/ctags-install-alpine.sh "$OUTPUT"
+cp -a ./dev/libsqlite3-pcre/install-alpine.sh "$OUTPUT/libsqlite3-pcre-install-alpine.sh"
+
+export OUTPUT
+./cmd/symbols/go-build.sh
+
+# Build either symbols or ctags (for dev)
+DOCKER_TARGET="${DOCKER_TARGET:-symbols}"
```

Comment:
Optional alternative, YMMV.

```suggestion
: ${DOCKER_TARGET:=symbols}
```